eZ/Publish/Core/Repository/SiteAccessAware/URLAliasService.php

```diff
@@ -96,4 +96,14 @@ public function load($id)
     {
         return $this->service->load($id);
     }
+
+    public function refreshSystemUrlAliasesForLocation(Location $location)
+    {
+        $this->service->refreshSystemUrlAliasesForLocation($location);
+    }
+
+    public function deleteCorruptedUrlAliases()
```

Comment:
This is new method. Can not we enforce return type hint as `int`?

Reply:
Fixed via b2f94d9